packages/cli/src/plugins/etherscan.ts

```diff
@@ -32,7 +32,8 @@ const apiUrls = {
   [42220]: 'https://api.celoscan.io/api',
   [44787]: 'https://api-alfajores.celoscan.io/api',
 }
-type ChainId = keyof typeof apiUrls
+
+export type EtherscanChainId = keyof typeof apiUrls
```

Comment:
Will also need to refactor the references below!

Reply:
whops, I did that commit on phone and totally missed it. I updated the references now